osu.Game/Screens/Backgrounds/BackgroundScreenBeatmap.cs

```diff
@@ -37,15 +50,19 @@ public WorkingBeatmap Beatmap
                         }
 
                         b.Depth = newDepth;
-                        AddInternal(Background = b);
+                        FadeContainer.Add(Background = b);
                         Background.BlurSigma = BlurTarget;
+                        FadeContainer.StoryboardReplacesBackground.BindTo(StoryboardReplacesBackground);
                     }));
                 });
             }
         }
 
         public BackgroundScreenBeatmap(WorkingBeatmap beatmap = null)
         {
+            FadeContainer = CreateFadeContainer();
+            InternalChild = FadeContainer;
+            EnableUserDim.BindTo(FadeContainer.EnableUserDim);
```

Comment:
In this case it doesn't matter, but I'd prefer this to be written as `FadeContainer.EnableUserDim.BindTo(EnableUserDim)`.

In the general sense, when `.BindTo(other)` happens, the values from `other` are instantly copied into the bindable, so this would mean the value from `FadeContainer.EnableUserDim` is coped into `EnableUserDim`.